### PR TITLE
Fix: Washable Magboots

### LINKED ISSUE #17168: CE Magboots turn into shoes when coloured in washing machine
**Issue Description**:
Advanced magboots turn into coloured shoes when you put them into a washing machine

**What did you expect to happen**: 
Nothing at all.

**What happened instead**:
The advanced magboots turned into red shoes until I turned them on or off.

**Why is this bad/What are the consequences**:
Stealth 100 advanced magboots theft.

**Steps to reproduce the problem**:
Put CE magboots in washing machine with crayon. Turn it on. Wait.

**When did the problem start happening**:
No idea.

**Extra information**:
Also apparently works on normal magboots.

### PR DESCRIPTION
**Pull request by tomasalves8**
## What Does This PR Do

Refactors the code for the Washing Machine and fixes #17168.

## Why It's Good For The Game

Codes looks is more readable this way. Magboots should not be washable anyway.

## Changelog
:cl:
fix: Washable Magboots
/:cl:

### CODE REVIEW

**Review by hal9000PR — CHANGES_REQUESTED** [review]
Remove the commented out code

**Comment by tomasalves8** [comment]
@hal9000PR

File: code/game/machinery/washing_machine.dm [hunk 1]
```diff
@@ -231,50 +231,25 @@
...
+			/obj/item/clothing/head/syndicatefake,
+			/obj/item/clothing/head/helmet,
+			/obj/item/clothing/gloves/furgloves,
+			/obj/item/clothing/gloves/color/black/krav_maga/sec,
+			/obj/item/clothing/shoes/magboots
+		)
+		for(var/X in unwashable)
+			if(istype(W,X))
+				to_chat(user, "<span class='warning'>This item cannot be washed.</span>")
+				return
```
Comment (hal9000PR) [inline code/game/machinery/washing_machine.dm:252]: there is a helper for this, `is_type_in_list`

File: code/game/machinery/washing_machine.dm [hunk 2]
```diff
@@ -231,50 +231,25 @@
...
-			to_chat(user, "This item does not fit.")
-			return
-		if( istype(W,/obj/item/clothing/gloves/furgloves ) )
-			to_chat(user, "This item does not fit.")
-			return
-		if(istype(W, /obj/item/clothing/gloves/color/black/krav_maga/sec))
-			to_chat(user, "<span class='warning'>Washing these gloves would fry the electronics!</span>")
-			return
+		
+		var/unwashable = list(
```
Comment (hal9000PR) [inline code/game/machinery/washing_machine.dm:235]:
```suggestion
		var/list/unwashable = list(
```

File: code/game/machinery/washing_machine.dm (hunk 1 above)
Comment (tomasalves8) [reply code/game/machinery/washing_machine.dm:252]: Cool

File: code/modules/admin/admin.dm [hunk 3]
```diff
@@ -367,7 +367,7 @@ GLOBAL_VAR_INIT(nologevent, 0)
 
 			if("Regular Restart")
 				var/delay = input("What delay should the restart have (in seconds)?", "Restart Delay", 5) as num|null
-				if(!delay)
+				if(!delay && delay != 0)
```
Comment (SteelSlayer) [inline code/modules/admin/admin.dm:370]: This seems like an unrelated change.

**Review by SteelSlayer — COMMENTED** [review]
You've made this PR with your `master` branch which will probably cause issues for any future PRs you make after this is merged. You might consider re-opening this using a separate branch.

File: code/modules/admin/admin.dm (hunk 3 above)
Comment (tomasalves8) [reply code/modules/admin/admin.dm:370]: It doesn't require a new PR itself, it's a minor fix.

**Comment by tomasalves8** [comment]
Yeah I need to get the habit of doing that. My next PR I will.

**Comment by dearmochi** [comment]
Please re-make this PR on a separate branch on your fork.
